linter_exclusions.yml

```diff
@@ -2175,18 +2175,58 @@ networkfabric controller create:
     workload_er_connections:
       rule_exclusions:
       - option_length_too_long
+    is_workload_management_network_enabled:
+      rule_exclusions:
+      - option_length_too_long
+networkfabric controller update:
+  parameters:
+    workload_er_connections:
+      rule_exclusions:
+      - option_length_too_long
+  rule_exclusions:
+  - missing_command_test_coverage
+networkfabric fabric update:
+  rule_exclusions:
+  - missing_command_test_coverage
 networkfabric rack create:
   rule_exclusions:
   - missing_command_test_coverage
 networkfabric rack delete:
   rule_exclusions:
   - missing_command_test_coverage
+networkfabric device create:
+  rule_exclusions:
+  - missing_command_test_coverage
 networkfabric device delete:
   rule_exclusions:
   - missing_command_test_coverage
+networkfabric interface create:
+  rule_exclusions:
+  - missing_command_test_coverage
 networkfabric interface delete:
   rule_exclusions:
   - missing_command_test_coverage
+networkfabric nni create:
+  parameters:
+    npb_static_route_configuration:
+      rule_exclusions:
+      - option_length_too_long
+    option_b_layer3_configuration:
+      rule_exclusions:
+      - option_length_too_long
+networkfabric nni update:
+  parameters:
+    npb_static_route_configuration:
+      rule_exclusions:
+      - option_length_too_long
+    option_b_layer3_configuration:
+      rule_exclusions:
+      - option_length_too_long
+  rule_exclusions:
+  - missing_command_test_coverage
+networkfabric l2domain update:
+  rule_exclusions:
+  - missing_command_test_coverage
```

Comment:
Same concern here

Reply:
Currently we are blocking the PATCH calls from user. Once PATCH is implemented completely, we will then unblock it.